server/datastore/mysql/policies_test.go

```diff
@@ -2277,3 +2280,105 @@ func testUpdatePolicyFailureCountsForHosts(t *testing.T, ds *Datastore) {
 	require.NoError(t, err)
 	require.Len(t, hostsUpdated, 0)
 }
+
+func testListGlobalPoliciesCanPaginate(t *testing.T, ds *Datastore) {
+	// create 30 policies
+	for i := 0; i < 30; i++ {
+		_, err := ds.NewGlobalPolicy(context.Background(), nil, fleet.PolicyPayload{Name: fmt.Sprintf("policy%d", i)})
+		require.NoError(t, err)
+	}
```

Comment:
Same as the other test, let's create some dummy policies that belong to a team (to confirm the filtering is working as expected).